core/src/main/res/values/strings.xml

```diff
@@ -748,39 +748,39 @@
-    <string name="sort_date_old_new">Date (Old \u2192 New)</string>
-    <string name="sort_duration_short_long">Duration (Short \u2192 Long)</string>
-    <string name="sort_duration_long_short">Duration (Long \u2192 Short)</string>
-
-    <string name="sort_a_z">A \u2192 Z</string>
-    <string name="sort_z_a">Z \u2192 A</string>
-    <string name="sort_new_old">New \u2192 Old</string>
-    <string name="sort_old_new">Old \u2192 New</string>
-    <string name="sort_short_long">Short \u2192 Long</string>
-    <string name="sort_long_short">Long \u2192 Short</string>
+    <string name="sort_title_a_z">Title (A - Z)</string>
+    <string name="sort_title_z_a">Title (Z - A)</string>
+    <string name="sort_date_new_old">Date (new - old)</string>
+    <string name="sort_date_old_new">Date (old - new)</string>
+    <string name="sort_duration_short_long">Duration (short - long)</string>
+    <string name="sort_duration_long_short">Duration (long - short)</string>
+
+    <string name="sort_a_z">A - Z</string>
+    <string name="sort_z_a">Z - A</string>
+    <string name="sort_new_old">New - old</string>
+    <string name="sort_old_new">Old - new</string>
+    <string name="sort_short_long">Short - long</string>
+    <string name="sort_long_short">Long - short</string>
 
     <!-- Rating dialog -->
     <string name="rating_title" tools:ignore="UnusedResources">Like AntennaPod?</string>
-    <string name="rating_message" tools:ignore="UnusedResources">We would appreciate it if you take the time to rate AntennaPod.</string>
-    <string name="rating_never_label" tools:ignore="UnusedResources">Leave me alone</string>
+    <string name="rating_message" tools:ignore="UnusedResources">Appreciate the work it took to create it by telling others.</string>
+    <string name="rating_never_label" tools:ignore="UnusedResources">No thanks</string>
     <string name="rating_later_label" tools:ignore="UnusedResources">Remind me later</string>
-    <string name="rating_now_label" tools:ignore="UnusedResources">Sure, let\'s do this!</string>
+    <string name="rating_now_label" tools:ignore="UnusedResources">Sure thing pal!</string>
 
     <!-- Share episode dialog -->
     <string name="share_dialog_include_label">Include:</string>
     <string name="share_playback_position_dialog_label">Playback position</string>
-    <string name="share_dialog_media_file_url_label">Media file address</string>
+    <string name="share_dialog_media_file_url_label">Media-file address</string>
     <string name="share_dialog_episode_website_label">Episode webpage</string>
     <string name="share_dialog_media_file_label">Media file</string>
 
     <!-- Audio controls -->
     <string name="audio_controls">Audio controls</string>
     <string name="playback_speed">Playback Speed</string>
     <string name="audio_effects">Audio Effects</string>
-    <string name="stereo_to_mono">Downmix: Stereo to mono</string>
+    <string name="stereo_to_mono">Downmix: Stereo in mono</string>
```

Comment:
Are you sure? The word is "to convert to", not "to convert in"

Reply:
My idea was to make it seem more like this is a two-channel track in two channels, as opposed to just reduced to one channel.
Then maybe "Downmix" can be omitted?